crates/core_simd/src/masks/bitmask.rs

```diff
@@ -120,39 +120,64 @@ where
     }
 
     #[inline]
-    #[must_use = "method returns a new array and does not mutate the original value"]
-    pub fn to_bitmask_array<const M: usize>(self) -> [u8; M] {
-        assert!(core::mem::size_of::<Self>() == M);
+    #[must_use = "method returns a new vector and does not mutate the original value"]
+    pub fn to_bitmask_vector(self) -> Simd<T, N> {
```

Comment:
maybe at least return `Simd<u8, N>`? saves a bunch of space for larger element types...

Reply:
not sure why I didn't do that from the start... done